README.md

```diff
@@ -127,7 +127,43 @@ The following optional parameters should be used in the `driver_config` for the
  - `poweron` - Power on the new virtual machine. Default: true
  - `vm_name` - Specify name of virtual machine. Default: `<suite>-<platform>-<random-hexid>`
  - `resource_pool` - Name of the resource pool to use when creating the machine. Will search first pool by default, can use value 'Resources' for none.
- - `clone_type` - Type of clone, will default to "full" to create complete copies of template. Needs a VM as template parameter, if "linked" clone desired.
+ - `clone_type` - Type of clone, will default to "full" to create complete copies of template. Other values: "linked", "instant"
```

Comment:
We usually specify the default value at the end of the line, e.g. "- Type of clone. Default: full (creates a copy of the template)." - @chef/docs-team will likely add a better suggestion here.

Reply:
That's a sensible suggestion, as I pulled out the detailed documentation of clone_type into a section by itself anyway.